### DISCUSSION TITLE
How can I login and tokens without any UI involvement ?

### DISCUSSION THREAD

**Discussion by ashwin-pandey** [ory/hydra#3741] [Q&A] [answered]
I have been trying to find a way/workaround to perform login and get tokens on the backend for developers and testers, because it requires browser involvement, I am not able to do so in my backend.

Is there any admin API that can help me in this case ?

Any help is much appreciated, thanks.

**Comment by zoopli** [comment, marked as answer]
curl can do anything a browser can, in my setup I login to kratos and do the whole consent flow just with curl in a script
you can probably do the same inside of postman
